admin/class-create-block-theme-admin.php

```diff
@@ -656,37 +656,96 @@ function create_admin_form_page() {
-				<?php _e('[Create a new child theme. The currently activated theme will be the parent theme.]', 'create-block-theme'); ?><br /><br />
-				<label><input value="clone" type="radio" name="theme[type]" class="regular-text code" onchange="document.getElementById('new_theme_metadata_form').removeAttribute('hidden');"/><?php _e('Clone ', 'create-block-theme'); echo wp_get_theme()->get('Name'); ?>
-				<?php _e('[Create a new theme cloning the activated theme. The resulting theme will have all of the assets of the activated theme as well as user changes.]', 'create-block-theme'); ?></label><br /><br />
-				<?php endif; ?>
-				<label><input value="save" type="radio" name="theme[type]" class="regular-text code" onchange="document.getElementById('new_theme_metadata_form').setAttribute('hidden', null);" /><?php _e('Overwrite ', 'create-block-theme'); echo wp_get_theme()->get('Name'); ?></label>
-				<?php _e('[Save USER changes as THEME changes and delete the USER changes.  Your changes will be saved in the theme on the folder.]', 'create-block-theme'); ?></label><br /><br />
-				<label><input value="blank" type="radio" name="theme[type]" class="regular-text code" onchange="document.getElementById('new_theme_metadata_form').removeAttribute('hidden', null);" /><?php _e('Create blank theme ', 'create-block-theme'); ?></label>
-				<?php _e('[Generates a boilerplate "empty" theme inside of this site\'s themes directory.]', 'create-block-theme'); ?></label><br /><br />
-
-				<div hidden id="new_theme_metadata_form">
-                    <p><?php _e('Items indicated with (*) are required.', 'create-block-theme'); ?></p>
-					<label><?php _e('Theme Name (*): The name of the theme.', 'create-block-theme'); ?><br />  <input type="text" name="theme[name]" class="regular-text" /></label><br /><br />
-					<label><?php _e('Theme Description: A short description of the theme.', 'create-block-theme'); ?><br /><textarea rows="4" cols="50" name="theme[description]" class="regular-text"></textarea></label><br /><br />
-					<label><?php _e('Theme URI: The URL of a public web page where users can find more information about the theme.', 'create-block-theme'); ?><br /><input type="text" name="theme[uri]" class="regular-text code" /></label><br /><br />
-					<label><?php _e('Author: The name of the individual or organization who developed the theme.', 'create-block-theme'); ?><br /><input type="text" name="theme[author]" class="regular-text" /></label><br /><br />
-					<label><?php _e('Author URI: The URL of the authoring individual or organization.', 'create-block-theme'); ?><br /><input type="text" name="theme[author_uri]" class="regular-text code" /></label><br /><br />
+				<div id="col-container">
+					<div id="col-left">
+						<div class="col-wrap">
+							<p><?php _e('Export your current block theme with changes you made to Templates, Template Parts and Global Styles.', 'create-block-theme'); ?></p>
```

Comment:
Let's add the theme name here?

Export your current block theme **[current-theme-name]** with changes you made to Templates, Template Parts and Global Styles.